docker/build.mk

```diff
@@ -60,22 +60,23 @@ dispatcher-image: base-image
     $(call cache_from,${BASE_TAG}/dispatcher-image) \
     docker/dispatcher-image
 
-define oss_fuzz_benchmark_template
-$(1)-project-name := $(shell cat benchmarks/$(1)/oss-fuzz.yaml | \
-                             grep project | cut -d ':' -f2 | tr -d ' ')
-$(1)-fuzz-target  := $(shell cat benchmarks/$(1)/oss-fuzz.yaml | \
+define benchmark_template
+$(1)-fuzz-target  := $(shell cat benchmarks/$(1)/benchmark.yaml | \
                              grep fuzz_target | cut -d ':' -f2 | tr -d ' ')
-$(1)-commit := $(shell cat benchmarks/$(1)/oss-fuzz.yaml | \
+$(1)-commit := $(shell cat benchmarks/$(1)/benchmark.yaml | \
                            grep commit: | cut -d ':' -f2 | tr -d ' ')
```

Comment:
commit and : can have space, so why not just search commit.

Reply:
This problem existed before my PR so I'm going to have to push back on fixing this. The PR touches a good deal of fuzzbench so there are many different things it doesn't change but that can be improved.